docs/learn-more/oauth.md

```diff
@@ -0,0 +1,87 @@
+# How to enable OAuth
+
+Following this guide, you will let the users to register and login using Google and Github OAUTH.
+
+- active `docker-compose.yml` (`~/.sourced/compose-files/__active__/docker-compose.yml`) should contain:
+    ```yml
+    version: '3.4'
+    services:
+    sourced-ui:
+      environment:
+        OAUTH_ENABLED_PROVIDERS: ${OAUTH_ENABLED_PROVIDERS}           # Comma separated list of available OAuth providers   (eg: github,google)
+        OAUTH_REGISTRATION_ROLE: ${OAUTH_REGISTRATION_ROLE}           # The role for the newly registered users using OAuth   'Admin'/'Alpha'/'Gamma'
+        OAUTH_GOOGLE_CONSUMER_KEY: ${OAUTH_GOOGLE_CONSUMER_KEY}       # OAuth provider consumer key (aka client_id)
+        OAUTH_GOOGLE_CONSUMER_SECRET: ${OAUTH_GOOGLE_CONSUMER_SECRET} # OAuth provider consumer secret (aka client_secret)
+        OAUTH_GITHUB_CONSUMER_KEY: ${OAUTH_GITHUB_CONSUMER_KEY}       # OAuth provider consumer key (aka client_id)
+        OAUTH_GITHUB_CONSUMER_SECRET: ${OAUTH_GITHUB_CONSUMER_SECRET} # OAuth provider consumer secret (aka client_secret)
+    ```
+
+In order to avoid some OAuth issues, make your instance of source{d} CE accessible from the Internet:
+- consider this example: `http://live.sourced.ce`
+    - `PROTOCOL` = `http`
+    - `HOST` = `live.sourced.ce`
+- you can use `ngrok` if running source{d} locally:
+    ```shell
+    $ ngrok http 8088
+    ```
+
+## For Google OAuth
+
+1. Configure your "OAuth consent screen"
+    - https://console.developers.google.com/apis/credentials/consent
+    - set up:
+        - "Application name"
+        - "Authorized domains": `${HOST}`
+1. Create a new "Credential"
+    - https://console.developers.google.com/apis/credentials
+    - create a new "Create OAuth client ID" for "Wab application"
+    - set up:
+        - "Application name"
+        - "Restrictions":
+            - Authorized JavaScript origins: `${PROTOCOL}://${HOST}`
+            - Authorized redirect URIs: `${PROTOCOL}://${HOST}/oauth-authorized/google`
```

Comment:
I think there was a way to restrict logins to the Google Apps domain (e.g. allow only `@sourced.tech` to log in). That's an interesting option to point out (not sure if Github has any kind of equivalent).